doc/news/_preparation_next_release.md

```diff
@@ -214,7 +214,9 @@ Many problems were resolved with the following fixes:
 
 - <<TODO>>
 - <<TODO>>
-- <<TODO>>
+- Haskell plugins and bindings are now correctly excluded when using BUILD_FULL or BUILD_STATIC
+  as this is currently unsupported. Another issue when building haskell plugins with a cached sandbox
```

Comment:
Which issue? These two points are unrelated, you should make two points out of it.